src/GlobalPayments.Api/Terminals/INGENICO/IngenicoVariables.cs

```diff
@@ -0,0 +1,122 @@
+﻿
+using System;
+
+namespace GlobalPayments.Api.Terminals.INGENICO {
+    internal class INGENICO_REQ_CMD {
```

Comment:
We're moving away from single files with all the enums/constants You should follow the naming and folder conventions of the other terminals in the SDK.